packages/host/app/routes/card.gts

```diff
@@ -1,29 +1,20 @@
 import Route from '@ember/routing/route';
-import type { ComponentLike } from '@glint/template';
 import { service } from '@ember/service';
 import ENV from '@cardstack/host/config/environment';
 import { parse } from 'qs';
 import type CardService from '../services/card-service';
 import type RouterService from '@ember/routing/router-service';
 import type LocalRealmService from '../services/local-realm';
-import Component from '@glimmer/component';
 import { Card } from 'https://cardstack.com/base/card-api';
 
 const { ownRealmURL, isLocalRealm } = ENV;
 const rootPath = new URL(ownRealmURL).pathname.replace(/^\//, '');
 
-class LocalRealmNotConnectedComponent extends Component {
-  <template>
-    Local realm not connected.
-  </template>
-}
-
 export interface Model {
-  card?: Card;
-  component?: ComponentLike<{ Args: {}; Blocks: {} }>;
+  card: Card;
```

Comment:
i wonder if the model could just more simply be `Card` instead of `{ card: Card; }`

Reply:
That does look simpler. I adjusted it to be `type Model = Card`.